tests/test_repo.py

```diff
@@ -470,19 +476,23 @@ def test_fork(self):
             }
     }
 
-        msg = GithubMessageV1(body={"body": body, "headers": headers })       
+        msg = GithubMessageV1(body={"body": body, "headers": headers })
         msg.validate()
         print(msg)
         print("printedd")
         assert msg.app_name == "Github"
-        assert msg.agent_name == "Demo-Webhook"
+        assert msg.agent_name == "fasUsernameExample"
         assert msg.event_name == "fork"
         assert msg.event_type == "repository"
-        assert msg.signature == "sha1=3b290912b53d4eb42a604b1a900e5818ab54f3df"
-        assert msg.signature_sha256 == "sha256=4449c05bc6e50e075c9962a04227527045ba0e85a04327a64d0aabf141497e19"
+        assert msg.signature == "sha1=3b290912b53d4eb42a604b1\
+            a900e5818ab54f3df"
+        assert msg.signature_sha256 == "sha256=4449c05bc6e50e075c9962a042\
+            27527045ba0e85a04327a64d0aabf141497e19"
         assert msg.target_id == "807808293"
-        assert msg.summary == "Demo-Webhook created fork on brngylni/demo_repo"
-        assert str(msg) == f"Event: Fork\nRepository: demo_repo (https://github.com/brngylni/demo_repo)\nForkee: demo_repo (https://github.com/Demo-Webhook/demo_repo)\nOwner: Demo-Webhook (https://github.com/Demo-Webhook)"
+        assert msg.summary == "fasUsernameExample created fork on brngylni/demo_repo"
+        assert str(msg) == "Event: Fork\nRepository: demo_repo (https://github.com/brngylni/demo_repo)\n\
```

Comment:
When strings are too long, wrap them in parenthesis:
```python
assert str(msg) == (
    "Event: Fork\n"
    "Repository: demo_repo (https://github.com/brngylni/demo_repo)\n"
    "Forkee: demo_repo (https://github.com/Demo-Webhook/demo_repo)\n"
    "Owner: Demo-Webhook (https://github.com/Demo-Webhook)"
)
```
Strings in parentheses will be automatically concatenated in Python.